.github/workflows/terraform.yaml

```diff
@@ -0,0 +1,35 @@
+---
+
+name: "Code Quality: Terraform"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  terraform:
+    name: Terraform
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1.2.1
```

Comment:
Thinking you could change this to `v1` (from `v1.2.1`) to bring it in line with all other version references.